packages/xdl/src/Web.js

```diff
@@ -1,9 +1,31 @@
+import fs from 'fs';
+import path from 'path';
 import opn from 'opn';
 import chalk from 'chalk';
 import Logger from './Logger';
 import * as UrlUtils from './UrlUtils';
 import { readConfigJsonAsync } from './project/ProjectUtils';
 
+function invokePossibleFunction(objectOrMethod, ...args) {
+  if (typeof objectOrMethod === 'function') {
+    return objectOrMethod(...args);
+  } else {
+    return objectOrMethod;
+  }
+}
+
+export function invokeWebpackConfig(env, argv) {
+  // Check if the project has a webpack.config.js in the root.
+  const projectWebpackConfig = path.resolve(process.cwd(), 'webpack.config.js');
```

Comment:
This should look for the config in the project root folder, `env.projectRoot`, not current working folder. You can run `expo start <full-path-to-project-root.` from anywhere and it should work.